translations/en-us.yml

```diff
@@ -598,7 +598,11 @@ node_card:
             withdrawn: Withdrawn
             archiving: Archiving
     tags: 'Tags:'
+    schema: 'Schema:'
```

Comment:
It doesn't look like we use "Schema" for user-facing language, but instead call it "Registration Type" so I'm checking with Mark if "Schema" is understandable, or if we should call it something else

Reply:
Looks like the language we want to use going forward is "Registration Template"

Reply:
+1